docs/content/installation/upgrade-with-helm.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Upgrading NGINX Ingress Controller with Helm
+description: "This document describes how to use Helm to upgrade NGINX Ingress Controller from 2.x and 3.0.x to the latest available release."
+weight: 1800
+doctypes: [""]
+toc: true
+docs: "DOCS-603"
+---
+
+## Background
+
+In NGINX Ingress Controller version 3.1.0, [changes were introduced](https://github.com/nginxinc/kubernetes-ingress/pull/3606) to Helm resource names, labels and annotations to fit with Helm best practices.
```

Comment:
Should we make this document part of our `helm upgrade` section in our existing doc?

https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/#upgrading-the-chart-1

Reply:
good shout, I can link this page from there. Having it separate makes it more visible imo.

Reply:
I agree that this document should be just a note in the upgrade section we already have. Or this should be a more general document on how to upgrade if we really want to have a separate one.